src/commands/aksDeployManifest/aksDeployManifest.ts

```diff
@@ -0,0 +1,145 @@
+import * as vscode from "vscode";
+import { Errorable, failed } from "../utils/errorable";
+import { getEnvironment, getReadySessionProvider } from "../../auth/azureAuth";
+import { selectClusterOptions } from "../../plugins/shared/clusterOptions/selectClusterOptions";
+import path from "path";
+import { ClusterPreference } from "../../plugins/shared/types";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { longRunning } from "../utils/host";
+import { invokeKubectlCommand } from "../utils/kubectl";
+import { getPortalResourceUrl } from "../utils/env";
+import { createTempFile } from "../utils/tempfile";
+import { checkExtension, handleExtensionDoesNotExist } from "../utils/ghCopilotHandlers";
+
+const GITHUBCOPILOT_FOR_AZURE_VSCODE_ID = "ms-azuretools.vscode-azure-github-copilot";
+
+export async function aksDeployManifest() {
+    // Check if GitHub Copilot for Azure extension is installed
+    const checkGitHubCopilotExtension = checkExtension(GITHUBCOPILOT_FOR_AZURE_VSCODE_ID);
+
+    if(!checkGitHubCopilotExtension) {
+        handleExtensionDoesNotExist(GITHUBCOPILOT_FOR_AZURE_VSCODE_ID);
+        return;
+    }
+
+    // Select manifest
+    const manifest = await getManifestFile();
+
+    if(manifest === undefined) { 
+        return;
```

Comment:
show a useful message to user that manifest files doesn't exist?

Reply:
This is shown in the getManifestFile function. This is what it looks like: 
<img width="384" alt="image" src="https://github.com/user-attachments/assets/411c00c1-6dc1-4d9c-a1f5-5072f25feb6a">

Reply:
💡 Just a side note: Please make sure the message is informative enough for user to know what action is intended, might be get some messaging which fills that gap.

Reply:
Added a little more description for clarity on action